### ISSUE TITLE
Move to Previous Paragraph: Shift+p shortcut not working with NVDA Version 2025.1 Beta 1

### ISSUE THREAD

**Issue by mystragical** [nvaccess/nvda#17977]
### Steps to reproduce:
Open any website containing articles using any browser.
Navigate bunch of times with p key while in browse-mode, so that you are on 2nd or 3rd paragraph.
Now, knowing that there are paragraphs prier to your current point, press shift+p.
NVDA doesn't move to previous paragraph, and a windows error notification sound is heard.
### Actual behavior:
It doesn't move to previous paragraph, just a windows notification sound is heard.
### Expected behavior:
NVDA should navigate to previous paragraph upon pressing shift+p.

#### NVDA installed/portable/running from source:
Portable.
#### NVDA version:
2025.1 beta 1

#### Name and version of other software in use when reproducing the issue:
Google chrome and edge, most recent public releases.
#### Other information about your system:
Windows 11, running most recent public release.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, the previous stable public releases work as expected.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes. 
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes.

**Comment by Adriani90** [comment]
Just to inform, I cannot reproduce this in NVDA alpha-36010,a9bcdffe (2025.2.0.36010)

**Comment by mystragical** [comment]
@Adriani90 , agree. I tried the portable version of latest alpha, and the issue is not there.
I'll wait for next update to beta or the RC and see and report if issue persists.

**Comment by SaschaCowley** [comment]
Welcome @mystragical ,

We are unable to process this issue as it stands.

- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.
- Pleas provide actual version numbers for the affected Chrome and Edge.

Kind Regards,
NV Access Software Developers

**Comment by SaschaCowley** [comment]
@mystragical are you still experiencing this issue?

**Comment by seanbudd** [comment]
Closing this issue. Please let us know if you're still experiencing it, and provide a full log file as requested
